src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicMobsMobTag.java

```diff
@@ -482,18 +666,77 @@ else if (mechanism.matches("level") && mechanism.requireInteger()) {
         // @tags
         // <MythicMobsMobTag.target>
         // -->
-        else if (mechanism.matches("target") && mechanism.requireObject(EntityTag.class)) {
-            EntityTag mTarget = mechanism.valueAsType(EntityTag.class);
-            if (mTarget == null || !mTarget.isValid() || mTarget.getLivingEntity() == null) {
+        tagProcessor.registerMechanism("target", false, EntityTag.class, (object, mechanism, target) -> {
+            if (target == null || !target.isValid() || target.getLivingEntity() == null) {
```

Comment:
This can probably use just `target.isLivingEnity()`, but should verify this still actually requires a living entity